source/Engine.cpp

```diff
@@ -586,15 +586,24 @@ void Engine::Step(bool isActive)
 		}
 	}
 
-	// Draw a highlight to distinguish the flagship from other ships.
+	outlines.clear();
+	const Color &cloakColor = *GameData::Colors().Get("cloak highlight");
```

Comment:
~We could even make this static, micro optimization!!!~

Reply:
I'm about 40% sure that making it static would decrease performance, as load/stores might not be in cache